docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -13,13 +13,14 @@ import { themes } from '@stardust-ui/react'
 import { ThemeContext } from '../../context/theme-context'
 import { constants } from 'src/lib'
 
+type ComponentMenuItem = { displayName: string; type: string }
+
 const pkg = require('../../../../package.json')
-const componentMenu = require('docs/src/componentMenu')
-const behaviorMenu = require('docs/src/behaviorMenu')
+const componentMenu: ComponentMenuItem[] = require('docs/src/componentMenu')
+const behaviorMenu: ComponentMenuItem[] = require('docs/src/behaviorMenu')
```

Comment:
May be we move `ComponentMenuItem` to `global.d.ts`?

```ts
const MenuItem = { displayName: string; type: string }

declare module 'componentMenu.json' {
  const value: ComponentMenuItem[]

  export default value
}
```

This will also allow us use `import` instead of `require`, I think.

Reply:
not sure we want `ComponentMenuItem` in global scope, it's not that relevant;

also, declaring the module like that would require lot more changes to fix the type errors that come out of it:
![screenshot 2018-11-12 at 13 38 43](https://user-images.githubusercontent.com/5442794/48348055-718bf480-e680-11e8-8bb4-3fc7d9b56657.png)

since this is not in the scope of the PR anyways, I decided not to fix, hope you're ok with that @layershifter